src/content/developers/docs/smart-contracts/languages/index.md

```diff
@@ -203,7 +203,7 @@ If you're new to Ethereum and haven't done any coding with smart contract langua
 **Yul**
 
 - Intermediate language for Ethereum.
-- Supports the [EVM](en/developers/docs/evm) and [Ewasm](https://github.com/ewasm), an Ethereum flavored WebAssembly, and is designed to be a usable common denominator of both platforms.
```

Comment:
Do me a favor, keep the leading slash here to keep it consistent with the rest of the codebase. =)
Also, looks like there is one more case of `en` needing removal on line 212:
should be
```
- Initially designed for an [optimistic rollup](/developers/docs/layer-2-scaling/#rollups-and-sidechains) contract.
```
Mind fixing that one up to?